src/pages/playground/Playground.tsx

```diff
@@ -346,9 +440,50 @@ const Playground: React.FC<PlaygroundProps> = props => {
   }, [editorSessionId]);
 
   const hash = isSicpEditor ? props.initialEditorValueHash : location.hash;
+  const { uuid } = useParams<string>();
```

Comment:
This is not how you use `useParams`. Please look at the documentation or other places where it's used in the codebase for the proper typing.